Dockerfiles/agent/datadog-kubernetes.yaml

```diff
@@ -10,8 +10,10 @@ config_providers:
 
 # Enable APM by setting the DD_APM_ENABLED envvar to true, or override this configuration
```

Comment:
this comment should be updated accordingly :)

Reply:
Indeed. Removed it.